addons/portal_rating/controllers/thread.py

```diff
@@ -0,0 +1,14 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.osv import expression
+from odoo.addons.mail.controllers import thread
+
+
+class ThreadController(thread.ThreadController):
+    def _get_fetch_domain(self, thread_model, thread_id, **kwargs):
+        domain = super()._get_fetch_domain(thread_model, thread_id, **kwargs)
+        if kwargs.get("portal") and kwargs.get("rating_value"):
```

Comment:
I don't think there is a need to restrict `rating_value` to portal only, if the key is provided, data should be sent.

Reply:
This rating_value is different from the rating_value in the messages. Actually, it's related to a missing feature (rating value cards since #121104) that I'm going to retrieve after this PR. This value represents a selected rating_value that the user wants to filter the messages with. I see the similar naming is confusing (I kept it as it was) so maybe we want to change this naming.